tests/components/hue/test_services.py

```diff
@@ -87,11 +85,8 @@ async def test_hue_activate_scene(hass: HomeAssistant, mock_api_v1) -> None:
 
 async def test_hue_activate_scene_transition(hass: HomeAssistant, mock_api_v1) -> None:
     """Test successful hue_activate_scene with transition."""
-    config_entry = config_entries.ConfigEntry(
-        version=1,
-        minor_version=1,
+    config_entry = MockConfigEntry(
```

Comment:
I wonder if this can be a fixture? It's all the same .

Reply:
I guess. The point of this PR is not really to improve the hue tests though, it's to not have to change hue tests when there are unrelated changes.

OK to merge as is, or so you want a fixture?